dokka-runners/dokka-gradle-plugin/src/main/kotlin/dokka/DokkaPublication.kt

```diff
@@ -47,7 +47,8 @@ constructor(
 
     abstract val moduleVersion: Property<String>
 
-    abstract val outputDir: DirectoryProperty
+    /** Output directory for the finished Dokka publication. */
+    abstract val outputDirectory: DirectoryProperty
 
     abstract val cacheRoot: DirectoryProperty
```

Comment:
AFAIK `cacheRoot` is not used inside dokka and so probably we don't need to expose it anywhere at all and just use some temp dir

Reply:
I agree it makes sense to remove it, but that should be done in the DokkaConfiguration. DGP is just a wrapper around Dokka's core configuration, and possibly other Dokka plugins use the cache dir.